### DISCUSSION TITLE
DietPi-Software | OCRmyPDF

### DISCUSSION THREAD

**Discussion by Nickname-T** [MichaIng/DietPi#6786] [Software Request]
Would be glad if OCRmyPDF could be added to the software library. 
A bonus could became easy to use GUI or similar. 

usecase: scan documents as pdf into a folder/share. OCRmyPDF should monitor the folder and automatically generate a searchable pdf.
Maybe another topic: some kind of software (you could name it KI) could try to understand the different files (content) to learn and later generate filenames and folder to organize the files automatically. 

Maybe this could be very nice as well, as it incl ocr as far as I can see:
DietPi-Software | Paperless-ngx #5489

**Comment by Joulinar** [comment]
Looks like an apt package that can easily installed using `apt install ocrmypdf`.
